liquidation-monitor/src/agent.spec.js

```diff
@@ -417,7 +428,11 @@ describe('handleBlock', () => {
 
     setPriceMocks(mockBtcPrice, mockBtcCollateralFactor, mockBtcCTokenRate);
     setPriceMocks(mockEthPrice, mockEthCollateralFactor, mockEthCTokenRate);
-    await (provideHandleBlock(initializeData))();
+
+    // Set the initial alert to 0 seconds and the first block time to 15 seconds
+    initializeData.nextAlertTime = 0;
```

Comment:
```suggestion
    // Set the first block time to 15 seconds
```

Reply:
Also, this file should be linted.  It looks like `mockUsdcCollateralFactor` and `mockUsdcCTokenRate` are never used.

Reply:
Ok, I can properly mock it out.

Reply:
Done! Addressed in commit eb58f1732afc2ab71e40882b9760a78a50a8a44a